### DISCUSSION TITLE
unable to authenticate in MongoDB 5.0

### DISCUSSION THREAD

**Discussion by jolmedo** [ClickHouse/ClickHouse#34508] [Q&A] [answered]
Hi, everyone. I saw the progression of the issue handling. I think and believe all is OK now. I just need you to tell me what to do in my deployments. I've just installed Clickhouse 21.8.9, and I'm trying to make some tests in order to extract from MongoDB and fill out an aggregatingMergeTree table in Clickhouse. I've been reading a lot of tech doc about Clickhouse possibilities, so I know that is not the only way to accomplish what I want to do. But it's a valid way, so I want to test it. My Clickhouse installation comes from downloaded DEB files (I'm using Ubuntu 20.04 in my laptop). It seems I have to re-compile Clickhouse? What do you advice me to do? Thanks in advance.

PS: I've just tried in a MongoDB 3.6 and a MongoDB 4.0, the outcome is the same: no admit empty username or cannot authenticate if I use credentials

**Comment by alexey-milovidov** [comment, marked as answer]
MongoDB 5 authentication is implemented in December.

**Reply by jolmedo** [reply]
SO, I assume previous MongoDB versions are a waste of time if I try to implement this feature. Thanks for your answer
